.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea
+.venv
+.env
```

Comment:
Consider adding entries to ignore database files like `.sqlite` or `.db3` to prevent them from being pushed to the repository.